packages/@ourworldindata/grapher/src/stackedCharts/StackedUtils.test.ts

```diff
@@ -72,5 +100,25 @@ describe(stackSeries, () => {
         expect(seriesArr[1].points[0].valueOffset).toEqual(0)
         const series = stackSeries(withMissingValuesAsZeroes(seriesArr))
         expect(series[1].points[0].valueOffset).toEqual(10)
+        expect(series[2].points[0].valueOffset).toEqual(12)
+    })
+})
+
+describe(stackSeriesInBothDirections, () => {
+    it("can stack positive values", () => {
+        const series = stackSeriesInBothDirections(
+            withMissingValuesAsZeroes(seriesArr)
+        )
+        expect(series[1].points[0].valueOffset).toEqual(10)
+        expect(series[2].points[0].valueOffset).toEqual(12)
```

Comment:
I took some time to grasp where these offsets are coming from; I think adding a comment here for each line will help future me :)
```suggestion
        expect(series[1].points[0].valueOffset).toEqual(10) // USA 2000
        expect(series[2].points[0].valueOffset).toEqual(12) // France 2000
```